source/includes/_workout_types.md

```diff
@@ -1,63 +1,88 @@
 # Workout Types
 
-The Wahoo Fitness API supports the following workout types:
-
-Id | Description
----------- | -------
-0	  |	BIKING
-1	  |	RUNNING
-2	  |	FE
-3	  |	RUNNING_TRACK
-4	  |	RUNNING_TRAIL
-5	  |	RUNNING_TREADMILL
-6	  |	WALKING
-7	  |	WALKING_SPEED
-8	  |	WALKING_NORDIC
-9	  |	HIKING
-10	|	MOUNTAINEERING
-11	|	BIKING_CYCLECROSS
-12	|	BIKING_INDOOR
-13	|	BIKING_MOUNTAIN
-14	|	BIKING_RECUMBENT
-15	|	BIKING_ROAD
-16	|	BIKING_TRACK
-17	|	BIKING_MOTOCYCLING
-18	|	FE_GENERAL
-19	|	FE_TREADMILL
-20	|	FE_ELLIPTICAL
-21	|	FE_BIKE
-22	|	FE_ROWER
-23	|	FE_CLIMBER
-24	|	SWIMMING
-25	|	SWIMMING_LAP
-26	|	SWIMMING_OPEN_WATER
-27	|	SNOWBOARDING
-28	|	SKIING
-29	|	SKIING_DOWNHILL
-30	|	SKIINGCROSS_COUNTRY
-31	|	SKATING
-32	|	SKATING_ICE
-33	|	SKATING_INLINE
-34	|	LONG_BOARDING
-35	|	SAILING
-36	|	WINDSURFING
-37	|	CANOEING
-38	|	KAYAKING
-39	|	ROWING
-40	|	KITEBOARDING
-41	|	STAND_UP_PADDLE_BOARD
-42	|	WORKOUT
-43	|	CARDIO_CLASS
-44	|	STAIR_CLIMBER
-45	|	WHEELCHAIR
-46	|	GOLFING
-47	|	OTHER
-49  | 	BIKING_INDOOR_CYCLING_CLASS
-56  |	WALKING_TREADMILL
-61	| 	BIKING_INDOOR_TRAINER
-62 	|	MULTISPORT
-63	|	TRIATHALON
-64	|	EBIKING
-65	|	TICKR_OFFLINE
-66	|	YOGA
-67	|	RUNNING_RACE
\ No newline at end of file
+The Wahoo Fitness API supports the following workout locations, families and types:
+
+Workout Type Locations:
+
+Id         | Description                  |
+---------- | ---------------------------- |
+0          |  INDOOR                      |
+1          |  OUTDOOR                     |
+
+
+Workout Type Families:
+
+Id         | Description                  |
+---------- | ---------------------------- |
+0          |  BIKING                      |
+1          |  RUNNING                     |
+2          |  SWIMMING                    |
+3          |  WATER SPORTS                |
+4          |  SNOW SPORTS                 |
+5          |  SKATING                     |
+6          |  GYM                         |
+9          |  WALKING                     |
+30         |  N/A                         |
```

Comment:
@JustinDunnWahoo can you check that this isn't supposed to be "UNKNOWN"?

Reply:
@aengesath Unknown for Families is anything that does not have an ID according to the docs https://wahooligans.atlassian.net/wiki/spaces/AR/pages/11259510786/Wahoo+Workout+Types

but I am going to add Unknown to the workout types list, its ID is 255 on production